api/v2/zz_generated.deepcopy.go

```diff
@@ -19,7 +19,7 @@ package v2
 import (
 	"github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
 	"github.com/devfile/devworkspace-operator/apis/controller/v1alpha1"
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
```

Comment:
if I remember correctly, this change needs to be reverted every time after update-dev-resources

Reply:
Right